src/inc-lib-variants.php

```diff
@@ -62,11 +70,13 @@ function lovd_fixHGVS ($sVariant, $sType = 'g', $sReference = '')
     // Check for a reference sequence.
     if ($sReference !== false) {
         if ($sReference == '') {
-            if (preg_match('/^[NX][CGMR]_[0-9]{6,9}\.[0-9]+:/', $sVariant)) {
+            if (preg_match('/^(\(?(LRG|ENS[GT]|[NX][CGMRTW])_?[0-9]+([.t][0-9]+)?\)?){1,2}:/', $sVariant)) {
```

Comment:
This regexp allows for invalid reference sequences to be passed. As the function doesn't do anything with the reference (`lovd_getVariantInfo()` checks it, I guess), maybe we don't need to be too strict about it. I'll have a look.